onadata/settings/docker.py

```diff
@@ -72,9 +72,9 @@
     ENKETO_API_TOKEN = 'abc'
     ENKETO_PROTOCOL = 'https'
     ENKETO_URL = 'https://enketo.ona.io/'
-    ENKETO_API_SURVEY_PATH = '/api_v1/survey'
+    ENKETO_API_SURVEY_ALL_PATH = '/api_v1/survey'
     ENKETO_API_INSTANCE_PATH = '/api_v1/instance'
-    ENKETO_PREVIEW_URL = urljoin(ENKETO_URL, ENKETO_API_SURVEY_PATH +
+    ENKETO_PREVIEW_URL = urljoin(ENKETO_URL, ENKETO_API_SURVEY_ALL_PATH +
```

Comment:
We might need this, no?

Reply:
We might need it if we do need it, I think our most used use case is to simply return an Enketo URL for data entry. There will be situations when we may need all or specific Enketo URLs. I will ascertain that we should be deliberate on how we retrieve the URLs. Is there a point on requesting a URL resource that we may not use?